packages/palette/src/elements/MultiSelect/tokens.ts

```diff
@@ -1,7 +1,11 @@
 import { themeGet } from "@styled-system/theme-get"
-import { css } from "styled-components"
+import { css, ExecutionContext } from "styled-components"
 import { State } from "../Select/types"
 
+interface MultiSelectProps extends ExecutionContext {
+  complete?: boolean
+}
+
 export const MULTISELECT_STATES: Record<State, any> = {
```

Comment:
can't we use the props here instead of `any` then we don't need it at line 40?

Reply:
Not too sure on this one... doesn't like it